imagersite/imager_profile/models.py

```diff
@@ -50,31 +51,17 @@ class ImagerProfile(models.Model):
     )
     job = models.CharField(default='', max_length=75)
     website = models.CharField(default='', max_length=255)
+    pub_pics = ImagerPhoto.objects.filter(published='PB').all().count()
```

Comment:
These attributes make migrating into a new database a pain because it queries the database before the database has been set up. Either keep all querying in the views or wrap these in model managers or class methods.